soc/mediatek/mt8xxx/mt8365/Kconfig.defconfig

```diff
@@ -0,0 +1,11 @@
+# Copyright 2023 The ChromiumOS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_MT8365
+
```

Comment:
this PR is not properly structured, this commit adds soc files and 1 board file, fix it so there is 1 commit adding the soc files, 1 commit adding the board files